birdhouse/components/jupyterhub/default.env

```diff
@@ -80,24 +80,25 @@ export JUPYTERHUB_ADMIN_USERS='{\"${MAGPIE_ADMIN_USERNAME}\"}'  # python set syn
 #  export JUPYTERHUB_RESOURCE_LIMITS="
 #     user:user1:mem_limit=30G
 #     group:group1:mem_limit=10G:cpu_limit=1
-#     group:group2:cpu_limit=3:gpu_ids=gpu1,gpu2,gpu3
-#     user:user2:gpu_ids=gpu1,gpu2,gpu3:gpu_count=2
+#     group:group2:cpu_limit=3:gpu_ids=0,3,4
+#     user:user2:gpu_ids=1,2,3:gpu_count=2
 #  "
 #
 #  Supported limits are: `mem_limit`, `cpu_limit`, `gpu_count`, `gpu_ids`. See the Jupyterhub Dockerspawner documentation
 #  for details and supported values for mem_limit and cpu_limit.
 #   - https://jupyterhub-dockerspawner.readthedocs.io/en/latest/api/index.html#dockerspawner.DockerSpawner.cpu_limit
 #   - https://jupyterhub-dockerspawner.readthedocs.io/en/latest/api/index.html#dockerspawner.DockerSpawner.mem_limit
-#  gpu_ids are a comma separated list of the GPU ids available on the host that you want to make available to
-#  the user or group. GPU ids can typically be discovered by running the `nvidia-smi` command.
+#  gpu_ids are a comma separated list of the GPU uuids or zero based indexes available on the host that you want to make available 
+#  to the user or group. GPU uuids and indexes can be discovered by running the `nvidia-smi --list-gpus` command.
```

Comment:
Again I am noob with gpus but should we provide an equivalent command (`nvidia-smi --list-gpus`) for AMD gpus?  To be neutral and not advertise for free for any specific company?

I know Nvidia is much more popular but is there any possibility that some one would be using AMD or that's an impossible scenario?